---
shipbot_ros/SpigotState state
